### DISCUSSION TITLE
Postgres scaling

### DISCUSSION THREAD

**Discussion by s-kris** [umami-software/umami#968] [Q&A]
Hello Mike,

Thank you so much for this product. Love the idea that the compute can be a nextjs app.

I have no experience in building analytics products earlier. So, pardon my question.

Is postgres performant enough once we have large amounts of data? I plan to use umami for my blogging platform. My understanding is that, typically, other analytics products use clickhouse for analytics data.

**Comment by mikecao** [comment]
Postgres performs very well, especially for a single self-hosted app. The reason others use clickhouse is because they need to support many customers on a single instance. Umami doesn't have that problem.

**Reply by s-kris** [reply]
Thanks for answering Mike.

Sorry if I was unclear. I want to use umami for my blogging platform for multiple customers. At the moment I have a very few customers and their pageviews are not too high.

So, I was just wondering how it scales as I go. :)

**Comment by mikecao** [comment]
Postgres should be able to scale fine. A lot of places use it as a production database. You only need to look at other scaling solutions when it becomes an issue, like millions of views per day.
